lib/src/Uncertainty/Algorithm/Simulation/SobolSimulationAlgorithm.cxx

```diff
@@ -384,7 +384,8 @@ Graph SobolSimulationAlgorithm::drawIndexConvergence(const UnsignedInteger margi
       dataUpperBound.add(pt);
     }
   }
-  const Curve estimateCurve(dataEstimate, "red", "solid", 2, "index estimate");
+  Curve estimateCurve(dataEstimate, "index estimate");
```

Comment:
This code is ok, but not the code below: `const Curve lowerBoundCurve(dataLowerBound, "green", "solid", 1, "bounds");`. 

![image](https://github.com/openturns/openturns/assets/31351465/a82c0dee-92ba-4a91-8186-a11285d81821)

Please use the second default color in the palette and use this color for both the lower and upper bounds.